plex/constants.py

```diff
@@ -79,8 +79,9 @@
 
 # media provider setting identifiers and values
 SETTINGS_PROVIDER_AUTHENTICATION = 'plex.authentication'
-SETTINGS_PROVIDER_AUTHENTICATION_OPTION_LOCAL = 0
-SETTINGS_PROVIDER_AUTHENTICATION_OPTION_MYPLEX = 1
+SETTINGS_PROVIDER_AUTHENTICATION_OPTION_LOCAL_NOAUTH = 0
+SETTINGS_PROVIDER_AUTHENTICATION_OPTION_LOCAL_TOKEN = 1
+SETTINGS_PROVIDER_AUTHENTICATION_OPTION_MYPLEX = 2
```

Comment:
Maybe we should make `MYPLEX = 0` because it seems to be the most common authentication method.